apps/frontend/src/network/__tests__/Routes.test.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import { render, waitFor, fireEvent } from '@testing-library/react';
+import { authTestUtils } from '@asap-hub/react-components';
+import nock from 'nock';
+import userEvent from '@testing-library/user-event';
+
+import Routes from '../Routes';
+import { API_BASE_URL } from '../../config';
+import TeamsResponse from '../../fixtures/teams.json';
+import UsersResponse from '../../fixtures/users.json';
+
+const renderNetworkPage = async (pathname: string, query = '') => {
+  const result = render(
+    <authTestUtils.Auth0Provider>
+      <authTestUtils.WhenReady>
+        <authTestUtils.LoggedIn user={undefined}>
+          <MemoryRouter initialEntries={[{ pathname, search: query }]}>
+            <Route path={'/network'}>
+              <Routes />
+            </Route>
+          </MemoryRouter>
+        </authTestUtils.LoggedIn>
+      </authTestUtils.WhenReady>
+    </authTestUtils.Auth0Provider>,
+  );
+  await waitFor(() =>
+    expect(result.queryByText(/auth0/i)).not.toBeInTheDocument(),
+  );
+
+  await waitFor(() =>
+    expect(result.queryByText(/Loading/i)).not.toBeInTheDocument(),
+  );
+  return result;
+};
+
+beforeEach(() => {
+  nock.cleanAll();
+  nock(API_BASE_URL, {
+    reqheaders: { authorization: 'Bearer token' },
+  })
+    .get('/teams')
+    .reply(200, TeamsResponse);
+
+  nock(API_BASE_URL, {
+    reqheaders: { authorization: 'Bearer token' },
+  })
+    .get('/users')
+    .reply(200, UsersResponse);
+});
+
+describe('Test toggle to team', () => {
+  it('Changes the placeholder', async () => {
+    const { getByText, queryByText, getByRole } = await renderNetworkPage(
+      '/network/teams',
+    );
+    const toggle = getByText('People');
+    const searchBox = getByRole('textbox') as HTMLInputElement;
```

Comment:
I *think* there's a `search` `role` that we should probably apply, if it's not already applied due to the input type, and test here using `getByRole('search')`

Reply:
Ill address it in the search integration ticket if thats cool?

Reply:
ya